ksqldb-common/src/main/java/io/confluent/ksql/util/KsqlConfig.java

```diff
@@ -311,9 +311,12 @@
   public static final String KSQL_SUPPRESS_BUFFER_SIZE_BYTES = "ksql.suppress.buffer.size.bytes";
   public static final Long KSQL_SUPPRESS_BUFFER_SIZE_BYTES_DEFAULT = -1L;
   public static final String KSQL_SUPPRESS_BUFFER_SIZE_BYTES_DOC =
-      "Bound the number of bytes that the "
-      + "buffer can use for suppression. Negative size means the buffer will be unbounded. If the "
-      + "buffer exceeds its max capacity, a StreamsException stating this is thrown";
+      "Config to control the maximum number of bytes used in the suppress buffer for a query.\n"
+      + "This can be overridden per query, from the CLI using the `SET` command.\n"
+      + "By default, the amount is unbounded, which can result in out of memory errors. If a "
+      + "bound is desired a good heuristic is to set the buffer max size to be equal to the heap "
+      + "memory divided by the max number of persistent queries. If the bound is exceeds its max "
+      + "capacity, a StreamsException stating this is thrown.";
```

Comment:
I prefer the previous, straight-to-the-point doc. I don't think config documentation should be prescriptive of _how_ someone should use the config, but rather descriptive of _what_ the config does. The `SET` command is well documented elsewhere, so I'm not sure it should be added here as well.

Reply:
Oh okay I will revert it to the previous docs, I was just trying emulate the style of the docs for some of the other configs I saw.